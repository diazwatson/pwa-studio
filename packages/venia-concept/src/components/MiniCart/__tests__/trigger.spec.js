import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Trigger from '../trigger';

const renderer = new ShallowRenderer();

test('renders the correct tree', () => {
    const props = {
        children: <span>Hi, I'm a child</span>
    };

    const tree = renderer.render(<Trigger {...props} />);

    expect(tree).toMatchSnapshot();
});

test('renders the correct tree when children not supplied', () => {
    const props = {};

    const tree = renderer.render(<Trigger {...props} />);

    expect(tree).toMatchSnapshot();
});
