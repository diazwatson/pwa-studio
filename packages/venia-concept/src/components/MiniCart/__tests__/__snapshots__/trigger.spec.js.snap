// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the correct tree 1`] = `
<button>
  <span>
    Hi, I'm a child
  </span>
</button>
`;

exports[`renders the correct tree when children not supplied 1`] = `<button />`;
